osu.Game/Screens/Play/HUD/HitErrorMeters/Bar/SidebarHitErrorMeter.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Rulesets.Judgements;
+using osu.Game.Rulesets.Scoring;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Screens.Play.HUD.HitErrorMeters.Bar
+{
+    public class SidebarHitErrorMeter : BarHitErrorMeter
```

Comment:
matching the capitalization with the inherited name
```suggestion
    public class SideBarHitErrorMeter : BarHitErrorMeter
```